static/src/js/components/CollectionAssociationPreviewProgress/CollectionAssociationPreviewProgress.scss

```diff
@@ -0,0 +1,21 @@
+.collection-association-preview-progress {
```

Comment:
I think some of this CSS looks funky but I don't want to hold this up. Would you mind writing a ticket to circle back and add a comment here?